src/include/firebird/FirebirdInterface.idl

```diff
@@ -448,7 +449,7 @@ interface Statement : ReferenceCounted
 								 uint itemsLength, const uchar* items,
 								 uint bufferLength, uchar* buffer);
 	uint getType(Status status);
-	const string getPlan(Status status, boolean detailed);
+	const string getPlan(Status status, isc_info_sql_plan_format plan_format);
```

Comment:
The build is failed because of this line:

> /home/appveyor/projects/firebird/gen/Release/cloop/release/bin/cloop /home/appveyor/projects/firebird/src/include/firebird/FirebirdInterface.idl pascal /home/appveyor/projects/firebird/src/include/gen/Firebird.pas Firebird --uses SysUtils \
	--interfaceFile /home/appveyor/projects/firebird/src/misc/pascal/Pascal.interface.pas \
    	--implementationFile /home/appveyor/projects/firebird/src/misc/pascal/Pascal.implementation.pas \
    	--exceptionClass FbException \
    	--functionsFile /home/appveyor/projects/firebird/temp/Release/func.pas \
    	--prefix I
/home/appveyor/projects/firebird/src/include/firebird/FirebirdInterface.idl:452:38: error: Interface/struct 'isc_info_sql_plan_format' not found.

Reply:
If I not mistaken, you have to add `isc_info_sql_plan_format` definition into /src/misc/pascal/Pascal.interface.pas

Also, I'm not sure it is legal to change interface method declaration, you should add new method instead.